src/olympia/blocklist/migrations/0037_alter_blockversion_block_type_and_more.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.16 on 2024-10-31 13:26
+
+from django.db import migrations
+import olympia.amo.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blocklist', '0036_blocklistsubmission_block_type_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='blockversion',
+            name='block_type',
+            field=olympia.amo.fields.PositiveTinyIntegerField(choices=[(0, 'Blocked'), (1, 'Restricted')], default=0, null=True),
+        ),
+        migrations.AlterField(
+            model_name='blocklistsubmission',
+            name='block_type',
+            field=olympia.amo.fields.PositiveTinyIntegerField(choices=[(0, 'Blocked'), (1, 'Restricted')], default=0, null=True),
```

Comment:
Tiny deviation but I don't think we need to make the blocklistsubmission block_type field nullable. It wasn't already nullable and we are not adding it so it should probably remain required on DB level since we also provide a default value. or?

https://github.com/mozilla/addons-server/blob/master/src/olympia/blocklist/migrations/0036_blocklistsubmission_block_type_and_more.py#L13-L17